ui/packages/statement/src/components/StatementsOverview.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useReducer, useEffect, useContext } from 'react'
 import { Select, Button, Modal } from 'antd'
+import dayjs from 'dayjs'
```

Comment:
If using dayjs, we need to declare it as a dependency in the statement/package.json else it increases the bundle size, considering statement has already depended on the moment.js, I prefer to use the moment.js currently (we will replace the moment.js by day.js later)

Reply:
actually day.js is already used @baurine

Reply:
ok, got! so let's just add dayjs to the package.json dependencies field, @Fullstop000